doc/man3/SSL_CONF_cmd.pod

```diff
@@ -533,6 +546,10 @@ B<KTLS>: Enables kernel TLS if support has been compiled in, and it is supported
 by the negotiated ciphersuites and extensions. Equivalent to
 B<SSL_OP_ENABLE_KTLS>.
 
+B<CertificateCompression>: use the certificate compression extension, enabled by
```

Comment:
Will need to be decided if this should be on by default.

Reply:
It depends if any compression algorithms are available, as well.